src/beach/data/questions/en.json

```
@@ -135,13 +135,13 @@
     "answers": {
       "a": "Normal. Nothing illegal.",
       "b": "Suspension to A3. Team A reduces by one the number of players. Free –throw for team B.",
-      "c": "Suspension to A3 and A5. Team A reduces by one the number of players by two. Free-throw for team B.",
+      "c": "Suspension to A3 and A5. Team A reduces the number of players by two. Free-throw for team B.",
       "d": "Suspension to the “responsible team official” of team A. Team A reduces by one the number of players. Free-throw for team B."
     }
   },
   {
     "id": "2.9",
-    "question": "While team B is in possession of the ball, official A of team B requests the Team time-out and the timekeeper whistles. Before the referee's time-out whistle, B3 shoots on goal and scores. At this moment the referee whistles for the Team time-out.",
+    "question": "While team B is in possession of the ball, official B0 of team B requests a Team time-out and the timekeeper whistles. Before the referee's time-out whistle, B3 shoots on goal and scores. At this moment the referee whistles for the Team time-out.",
```

Im Handball werden die Offiziellen mit Buchstaben von A bis Z versehen. Offizieller A ist immer der Mannschaftsverantwortliche, deswegen sollten wir das hier bei A lassen. 

```suggestion
    "question": "While team B is in possession of the ball, official A of team B requests a Team time-out and the timekeeper whistles. Before the referee's time-out whistle, B3 shoots on goal and scores. At this moment the referee whistles for the Team time-out.",
```